content/browser-automation.md

```diff
@@ -114,26 +114,20 @@ More information on [Selenium Webdriver](http://docs.seleniumhq.org/projects/web
 
 # Browser Automation Tools for Java
 
-Some browser automation tools are specific to Java.
-
 ## Serenity BDD
 
+{{% text "java" %}}
```

Comment:
Afaik Serenity is java only, so I understand marking the whole thing. Do we provie an alternate text for other languages?

Reply:
I think we should break these tool descriptions out into separate files e.g. `tools\java\serenity-bdd` and just have the link in here. Something for future work on this file

Reply:
Agreed. The problem is the way the syntax works. You can't wrap syntax over `##` The build process complains. If you see my 3rd commit (You'll notice it failed the build), this was the reason why.

Reply:
Yes, I am aware ;) I'll make a note and fix separately.

Reply:
Still thinking that as the rest of the page is polyglot it would be ok to have the actual text for Serenity marked as "java" text blok, but have an alternate text for Ruby, javascript with something like "Serenity is jvm only" or something. And vice versa for the Ruby tools. (to prevent having too many in depth pages that would then loose context as @mattwynne mentioned in #151